lib/errorhandling.js

```diff
@@ -14,7 +14,7 @@ function renderError( err, res ) {
   });
 }
 
-module.exports.errorHandler = function(err, req, res) {
+module.exports.errorHandler = function(err, req, res, next) {
```

Comment:
It needs 4-args vs. 3 in order for express to treat it as error handling middleware, see http://expressjs.com/guide/error-handling.html